README.md

```diff
@@ -51,26 +36,27 @@ commercetools sync is a Java library that imports commercetools platform data in
 1. Synchronise data coming from an external system in any form (CSV, XML, etc..) that has been already mapped to 
 [commercetools-jvm-sdk](https://github.com/commercetools/commercetools-jvm-sdk) resource draft objects 
 (e.g. [CategoryDraft](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/categories/CategoryDraft.java)).
+    - The amount of data that needs to be synced between your external system and a commercetools project can be huge. To assure good performance we strongly recommend implementing a **delta sync** approach for your logic instead of **full sync**; means that only resources
+      which have been modified since the last time the sync has run need to be synced.
```

Comment:
> commercetools sync is a Java library which allows imports commercetools platform data in the following ways

Create you own event or cronjob based application and use the library to transform any external data (JSON, CSV, XML, REST API, DB, ...) into [commercetools-jvm-sdk](https://github.com/commercetools/commercetools-jvm-sdk) resource draft objects (e.g. [CategoryDraft](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/categories/CategoryDraft.java)) and import those into commercetools project.

**Note**:
 - It is often more efficient if you can setup your external data source to provide you only the changes (deltas) instead of the full data set on every import iteration.
 - There is dockerized ready-to-use CLI application [commercetools-project-sync](https://github.com/commercetools/commercetools-project-sync) which based on this library can synchronize entire data catalogue between the 2 commercetools projects.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/758/commits/818bf4d19712e0b4980113d5a2a02155e2a8192e